releasenotes/notes/handle-deferred-close-network-protocols-63a9e2c0374f0dd6.yaml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
Do we really need a security release note for a fix in some test util code ?

Reply:
I agree we can remove it